src/NetworkChest.lua

```diff
@@ -609,6 +609,62 @@ function M.onTick()
   M.update_queue()
 end
 
+function M.onTick_60()
+  M.updatePlayers()
+  M.check_alerts()
+end
+
+function M.handle_missing_material(entity, name)
+  -- did we already transfer something for this ghost/upgrade?
+  if GlobalState.alert_transfer_get(entity.unit_number) ~= true then
+    -- We can only do something about entities in a network
+    -- REVISIT: assuming "player" force only
+    local net = entity.surface.find_logistic_network_by_position(entity.position, "player")
```

Comment:
What happens if there are multiple logistic networks in the surface and the player is standing in the logistic network that doesn't need items?

Reply:
You are talking about a entity that has coverage from multiple "construction" logistic networks zone for cells that do not connect via "logistic" robots.

I assume it wouldn't show an alert in that case, as the request would be satisfied by the network that has the item.
The logistic network is based on the ghost position, not the player

If there are multiple networks that have coverage and none have the item, then it shouldn't matter which network gets the item.

I suppose there are corner cases where one network has construction robots and the other doesn't.
In that case, if `find_logistic_network_by_position()` return the network with no robots, then nothing will get added to that network and the alert will stay active.

Reply:
Got it, thanks for explaining. I tried out this branch and it makes a lot more sense.